### PR TITLE
Auto retry

### PR DESCRIPTION
**Pull request by laurence-hudson-tessella**
See example output: https://gist.github.com/laurence-hudson-tessella/61a6ea5f0bc3e91a43ab
(model runs 6/7/8/9 in staging db)

### CODE REVIEW

File: src/ModelWrapper/src/uk/ac/ox/zoo/seeg/abraid/mp/modelwrapper/model/ModelRunTemplate.ftl [hunk 1]
```diff
@@ -11,176 +11,234 @@
...
+    return(result)
+}
+
+attempt_model_run_with_clean_environment <- function() {
+    print("Before")
+    print(ls())
+    print(ls(envir = globalenv()))
+    print(search())
+
+    # Replace the environment of the attempt_model_run function, too ensure runs are completely independant
```
Comment (helena-patching-tessella) [inline src/ModelWrapper/src/uk/ac/ox/zoo/seeg/abraid/mp/modelwrapper/model/ModelRunTemplate.ftl]: to ensure runs are completely independent

File: src/ModelWrapper/src/uk/ac/ox/zoo/seeg/abraid/mp/modelwrapper/model/ModelRunTemplate.ftl [hunk 2]
```diff
@@ -11,176 +11,229 @@
...
+# Attempt the model run
+result <- attempt_model_run_with_clean_environment()
+
+if (result == 1) {
+    # Set up for a 2nd attempt at running the model
+    clean_up_model_run()
 
-# Delete temp dir for intermediate rasters
-unlink("temp", TRUE)
+    # Reattempt try the model run
```
Comment (edwiles) [inline src/ModelWrapper/src/uk/ac/ox/zoo/seeg/abraid/mp/modelwrapper/model/ModelRunTemplate.ftl]: Remove "try" ?

**Comment by edwiles** [comment]
LGTM with minor comment

**Comment by helena-patching-tessella** [comment]
LGTM

**Pull request merged**